src/components/common/FileUploadPreview.vue

```diff
@@ -83,29 +83,38 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
   //  is using a different validation approach or updating all components to use vuelidate.  Have decided
   //  to do this for the time being.
   private async validateFileInput (file: File): Promise<boolean> {
-    this.customErrorMessages = []
+    // clear any existing error messages
+    this.setCustomErrorMessage(null)
+
+    // validate the form (using rules)
     const isValid = this.$refs.form.validate()
-    // only perform page size validation when other validation has passed
-    if (isValid && file) {
```

Comment:
If `isValid` was false then the code fell through to line 108 and returned False. So I just simplified this.